src/ol/VectorRenderTile.js

```diff
@@ -81,6 +87,11 @@ class VectorRenderTile extends Tile {
      */
     this.getSourceTiles = getSourceTiles.bind(undefined, this);
 
+    /**
+     * @type {!function():void}
+     */
+    this.removeSourceTiles = removeSourceTiles.bind(undefined, this);
```

Comment:
Not that important, but I'm curious why `bind` here instead of calling like `removeSourceTiles(this)` below? Was this a typescript thing?

Reply:
Only to make it consistent with `getSourceTiles()`. But you're right, it adds unnecessary overhead. I'm going to add a commit to change as you suggest.